### DISCUSSION TITLE
Android Auto for Porsche

### DISCUSSION THREAD

**Discussion by Cay718** [Mr-MIBonk/M.I.B._More-Incredible-Bash#162] [Q&A] [answered]
I want to add Android Auto to my Porsche.  It has the POG11:
MHI2_ER_POG11_P4172
MU Software: 1119

The car already has CarPlay, USB support and Navigation activated.  Do I use the ALL-IN-ONE Patch or just try to add Android Auto?

**Comment by Cay718** [comment]
@harman-f  Can you provide some guidance on this question - Many thanks!

**Reply by harman-f** [reply]
Use M.I.B to patch and add AA image. 
afterwards you also can install latest maps.

**Reply by Cay718** [reply]
Hi @harman-f  - I successfully installed M.I.B on the MHi2 unit in my Porsche Cayman and the unit works fine and I can access the GEM, I executed the backup as per the instructions.  

However I am unsure of the exact steps to enable Android Auto.
This is my understanding:

Access menu items:
1.
=>m.i.b<=/patch_ifs_root/carplay_activation/check_coding
Should show:
Android Auto                ON
CarPlay                        ON
MirrorLink                    ON
USB Configuration      iPod - CarPlay and Android Auto

2.
=>m.i.b<=/patch_ifs_root/carplay_activation/usb_settings
USB Port 1 - standard
USB Port 2 - common
Both selected, if not select port 1 and 2 (the car has two USB ports), save changes and reboot unit

3.
=>m.i.b<=/patch_ifs_root/carplay_activation
Then press "Flash patched image | add FECs | CP/AA coding | and more"

Please could you correct it if I've got it wrong.

Thanks in advance!

**Reply by harman-f** [reply, marked as answer]
M.I.B. -> PATCH AIO

Will execute all the things you need to enable AA. Nothing more is needed.

**Reply by Cay718** [reply]
Patch and image installed.  Android Auto work as expected.
Many thanks @harman-f

**Comment by Bossweb2K** [comment]
> ALL-IN-ONE



> I want to add Android Auto to my Porsche. It has the POG11: MHI2_ER_POG11_P4172 MU Software: 1119
> 
> The car already has CarPlay, USB support and Navigation activated. Do I use the ALL-IN-ONE Patch or just try to add Android Auto?

hi mate!
Have you tried running the "SVM fix" function? This function will remove Software Version Management error from 5F (this "error" is a result of any FW Update you run on unit)

![173446239-1053ff99-6498-4745-96b1-ee0aa8b9a3b4](https://user-images.githubusercontent.com/51960463/236642419-0b603a9d-9ac1-4011-98ec-2d8275189e30.png)

**Reply by rbakjaji** [reply]
Can anyone help. I tried installing MIB 3.0 on my porsche macan. When i try to update using sd card 1 it says no files available for update.

**Reply by tunatoksoz** [reply]
Try this
https://github.com/LawPaul/MH2p_SD_ModKit

**Comment by ffichman** [comment]
How would I enable Android and wireless CarPlay?

**Reply by tunatoksoz** [reply]
Check this - it's now working.
https://github.com/LawPaul/MH2p_SD_ModKit
